package.json

```diff
@@ -29,7 +29,7 @@
         "lerna": "2.4.0",
         "fs-extra": "8.1.0",
         "capitalize": "^2.0.2",
-        "ovsx": "latest",
+        "ovsx": "^0.1.0-next",
```

Comment:
I spoke with @paul-marechal and since the dependency resolves to `0.1.0` and `next` should not be actual production dependencies we should use, we can simply reference `^0.1.0` as a version range. If ovsx publishes a proper patch version (ex: `0.1.1`) we will pick it up.

Reply:
Done, it makes sense, thanks for the clarification!